cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -258,6 +262,25 @@ def is_displayed(self):
         )
 
 
+class ButtonGroupDetailView(ServicesCatalogView):
+    title = Text("#explorer_title_text")
+
+    text = SummaryFormItem(
+        "Basic Information",
+        "Text",
+        text_filter=lambda text: re.sub(r"\s+Display on Button\s*$", "", text),
+    )
+    hover = SummaryFormItem("Basic Information", "Hover Text")
```

Comment:
Optional: why not call this variable `hover_text`?

Reply:
I like `hover`. base custom button implementation using same. :)